dhall/src/Dhall/Repl.hs

```diff
@@ -398,26 +399,91 @@ cmdQuit _ = do
   liftIO (putStrLn "Goodbye.")
   liftIO (throwIO Interrupt)
 
+help
+  :: ( Haskeline.MonadException m, MonadFail m, MonadIO m, MonadState Env m )
+  => HelpOptions m -> [String] -> m ()
+help hs _ = do
+  liftIO (putStrLn "Welcome to the Dhall REPL.")
```

Comment:
I think this would be better in the greeting than in the help output.

Maybe we could have a big Dhall in ASCII art?

```
  ____  _           _ _ 
 |  _ \| |__   __ _| | |
 | | | | '_ \ / _` | | |
 | |_| | | | | (_| | | |
 |____/|_| |_|\__,_|_|_|               
```
There are many other styles available here if that's something we might be interested in:
http://patorjk.com/software/taag/#p=display&f=Standard&t=Dhall

Reply:
Moved to greeting with version. Personally, I prefer regular text so as not to increase scrollback too much.

Reply:
I saw the colourful ASCII art of the Unison language REPL yesterday and thought it was nice and fun. But I don't have any strong preferences for having one of those in the Dhall REPL.